fs/mount/fs_automount.c

```diff
@@ -551,6 +862,37 @@ FAR void *automount_initialize(FAR const struct automount_lower_s *lower)
       ferr("ERROR: Failed to schedule work: %d\n", ret);
     }
 
+#ifdef CONFIG_FS_AUTOMOUNTER_DRIVER
+
+  /* Initialize the new automount driver instance */
+
+  nxsem_init(&priv->exclsem, 0, 1);
+
+  devpath = kmm_zalloc(sizeof(CONFIG_FS_AUTOMOUNTER_VFS_PATH) +
```

Comment:
Personally I do not like this way, but can't find a better solution without dynamic allocation.

Reply:
Another way is use PATH_MAX

Reply:
Done